core/src/main/java/com/gamification/core/CoreApplication.java

```diff
@@ -0,0 +1,14 @@
+package com.gamification.core;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CoreApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(CoreApplication.class, args);
+        System.out.println("Hi");
```

Comment:
its seems like application run check please remove it!

Reply:
I was for test and I had to delete this statment.